src/app/public/modules/inline-form/types/inline-form-button-action.ts

```diff
@@ -0,0 +1,5 @@
+export type SkyInlineFormButtonAction =
```

Comment:
Should we make this a proper enum over strings? @Blackbaud-SteveBrush thoughts?

Reply:
I'm torn. I'm assuming Alex is using strings so that they can easily be translated to CSS class names in the template. String enums don't exist in older versions of TypeScript (we need to support older versions for now), so we can either use number enums, or string "types" like @Blackbaud-AlexKingman is doing here. In the past, I've used a combination of number enums and a public `get actionName` to return a string representation to be used by the template for CSS classnames (until we can officially use string enums in a breaking change).

Reply:
On further discussion, we decided to leave as-is.